lib/editor/components/HourMinuteInput.js

```diff
@@ -92,7 +99,15 @@ export default class HourMinuteInput extends Component<Props, State> {
           onChange={this._onChange}
           onFocus={this._onFocus}
           placeholder={'mm'}
-          style={{ width: '6.5ch', marginLeft: -5, borderLeft: 'none', borderRadius: 0, ...style }}
+          style={{
+            width: '6.5ch',
+            marginLeft: -5,
+            borderLeft: 'none',
+            borderRadius: 0,
+            borderTopRightRadius: standaloneInput ? 3 : 0,
```

Comment:
Sort CSS

Reply:
Sorted in https://github.com/ibi-group/datatools-ui/pull/892/commits/69ed91b487e9585118c726639dd711c98d0c2df1